packages/card/components/Card.tsx

```diff
@@ -14,6 +20,14 @@ export interface CardProps extends React.HTMLProps<HTMLDivElement> {
    */
   paddingSize?: SpaceSize;
   children?: React.ReactNode | string;
+  /**
+   * The card header will be set to the top section of the card optionally. Set a header image along with corresponding alt text. Size can be set to "s", "m", or "l" to increase the header height.
+   */
+  header?: {
+    headerImg: string;
+    headerImgAltText?: string;
```

Comment:
Shouldn't we make this mandatory if the `headerImg` also is mandatory?

Reply:
IMO no, sometimes an image might be more of a background or decorative image that doesn't require additional context for screen readers. When a screen reader comes across null alt text, it will completely skip over the image, without announcing its presence.

Reply:
Thanks for clarifying, I wasn't sure how this will be used so I just assumed an image has a meaning. 👍